egs2/TEMPLATE/tts1/tts.sh

```diff
@@ -9,125 +11,132 @@ set -o pipefail
 
 log() {
     local fname=${BASH_SOURCE[1]##*/}
-    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $@"
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
 }
 help_message=$(cat << EOF
-Usage: $0 --train-set <train_set_name> --dev-set <dev_set_name> --eval_sets <eval_set_names> --srctexts <srctexts >
+Usage: $0 --train-set <train_set_name> --dev-set <dev_set_name> --eval_sets <eval_set_names> --srctexts <srctexts>
 
 Options:
+    --stage (int): Processes starts from the specifed stage.
+    --stop-stage (int): Processes is stopped at the specifed stage.
+    --ngpu (int): The number of gpus
     --nj (int): The number of parallel jobs
     --decode-nj (int): The number of parallel jobs for decoding
-    --ngpu (int): The number of gpus
     --gpu-decode (bool): Use gpu for decoding.
-    --stage (int): Processes starts from the specifed stage.
-    --stop-stage (int): Processes is stopped at the specifed stage.
-    --nlsyms: non-linguistic symbol list
-    --oov (str): Out of vocabrary symbol. The default is "<unk>"
 EOF
 )
 SECONDS=0
 
 # general configuration
-stage=1
-stop_stage=100
-ngpu=0       # number of gpus ("0" uses cpu, otherwise use gpu)
-nj=50         # numebr of parallel jobs
-decode_nj=50
-gpu_decode=false
-
-
-## The options for feature_extract
-audio_format=flac
-fs=16000
-feats_type=fbank
-fmax=""       # maximum frequency
-fmin=""       # minimum frequency
-n_mels=80     # number of mel basis
-n_fft=1024    # number of fft points
-n_shift=256   # number of shift points
-win_length="" # window length
-
-
-# exp tag
-tag="" # tag for managing experiments.
-exp=exp
-# config files
-train_config=
-train_args=
-
+stage=1          # stage to start.
+stop_stage=100   # stage to stop.
```

Comment:
Is it better to write the help comment in ${help_message}, isn't it? Though I didn't write both...

Reply:
Right. I also updated help message.